src/js/video.js

```diff
@@ -6,8 +6,10 @@ import VideoInfo from './info';
 import Playlist from './playlist';
 
 function eventListener(video, ev) {
+
 	// Dispatch progress event at start, 25%, 50%, 75% and 100%
-	if (ev.type === 'progress' && video.getProgress() % 25 !== 0) {
+	// If allProgress is set to true, then we send spoor events for every native video progress event (every 5 sec)
+	if (!video.allProgress && ev.type === 'progress' && video.getProgress() % 25 !== 0) {
```

Comment:
Can we have a test for this new option? I also think this is currently incorrect, I believe the property will be located at `video.opts.allProgress`.